src/ucp/api/ucp_def.h

```diff
@@ -80,6 +79,30 @@ typedef struct ucp_config                ucp_config_t;
  */
 typedef struct ucp_ep                    *ucp_ep_h;
 
+/**
+ * @ingroup UCP_ENDPOINT
+ * @brief Callback to process incomeing active message
+ *
+ * When the callback is called, @a flags indicates ho w @a should be handled.
+ *  
+ * @param [in]  arg     User-defined argument.
```

Comment:
maybe add a little verbiage to link with arg supplied in callback function registration